examples/theme_plugin.py

```diff
@@ -0,0 +1,33 @@
+"""Very simple theme plugin example providing simple color scheme changes."""
```

Comment:
i wonder if we should put this in a new folder `examples/plugins`? As it's a little more for reference than for running, as if i run example nothing happens as no viewer is launched. Or maybe there's a way to launch viewer and show plugin being used in same script too?

Reply:
Since I am still making changes, I haven't worked out a good example yet. I think its a good idea to have sub-category of examples specifically for plugins. Once I iron out the kinks I will add one in.